modules/apps/dynamic-data-mapping/dynamic-data-mapping-uad/src/main/java/com/liferay/dynamic/data/mapping/uad/constants/DDMFormInstanceUADConstants.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.uad.constants;
+
+/**
+ * @author Marcos Martins
+ */
+public class DDMFormInstanceUADConstants {
+
+	public static final String DEFAULT_DATE_FORMAT = "MMM dd yyyy 'at' HH:mm";
```

Comment:
@marcosapmf, the default date format constant is related to the form instance records, right? If so, can you please make it more explicity in the constant name?